build.gradle

```diff
@@ -1,34 +0,0 @@
-buildscript {
-    ext.kotlin_version = '1.3.70'
-    repositories {
-        jcenter()
-        google()
-        maven {
-            url "http://oss.sonatype.org/content/repositories/snapshots/"
-        }
-    }
-    dependencies {
-        classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0'
-        classpath 'com.android.tools.build:gradle:3.6.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.10.0"
-    }
-}
-
-subprojects {
-    repositories {
-        jcenter()
-        google()
-    }
-
-    apply plugin: 'checkstyle'
-
-    checkstyle { -> rootProject
-        configFile rootProject.file('config/checkstyle/checkstyle.xml')
-    }
-}
-
-wrapper {
-    gradleVersion = '5.6.4'
-}
```

Comment:
Got rid of this, not sure if needed. If it is, need to check what is the .gradle.kts equivalent of this